src/lib/services.json

```diff
@@ -1,17 +1,6 @@
 {
-  "github": {
-    "tokens": {
-      "token": "ONEGRAPH_GITHUB_TOKEN"
-    }
-  },
-  "spotify": {
-    "tokens": {
-      "token": "ONEGRAPH_SPOTIFY_TOKEN"
-    }
-  },
-  "salesforce": {
-    "tokens": {
-      "token": "ONEGRAPH_SALESFORCE_TOKEN"
-    }
-  }
+  "gitHub": null,
```

Comment:
There's not really much point doing this in a JSON file at this point. Just define the type in the d.ts.